src/App.jsx

```diff
@@ -1,104 +1,25 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+const posts = postsFromServer.map(post => {
+  const updatedPost = {
+    ...post,
+    comments:
+      commentsFromServer.filter(comment => comment.postId === post.id) || null,
+    user: usersFromServer.find(user => user.id === post.userId) || null,
```

Comment:
The use of `|| null` is unnecessary here. The `find` method will return `undefined` if no user is found, which is a more appropriate value than `null` for a non-existent user.